2010-05-26  Aurelien Jarno  <aurelien@aurel32.net>

	* posix/regex_internal.h: include <sys/param.h> to get MAX 
	definition.

diff --git a/posix/regex_internal.h b/posix/regex_internal.h
index 65a9905..d83b9df 100644
--- a/posix/regex_internal.h
+++ b/posix/regex_internal.h
@@ -26,6 +26,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <sys/param.h>
 
 #if defined HAVE_LANGINFO_H || defined HAVE_LANGINFO_CODESET || defined _LIBC
 # include <langinfo.h>
